README.md

```diff
@@ -37,28 +37,113 @@ services:
         }
 ```
 
+## Tips
+
+Plan [outputs](https://www.terraform.io/intro/getting-started/outputs.html)
+are published as sensors prefixed with `tf.output`. Use this to communicate
+information about the infrastructure created by Terraform to other
+components of the blueprint via Brooklyn's
+[dependent configuration](https://brooklyn.apache.org/v/0.9.0/yaml/yaml-reference.html#dsl-commands).
+
+For example, to attach a `TomcatServer` to an AWS security group that was
+created by a Terraform plan:
+
+```yaml
+services:
+
+- type: org.apache.brooklyn.entity.webapp.tomcat.TomcatServer
+  location:
+    jclouds:aws-ec2:us-west-2:
+      osFamily: centos
+      templateOptions:
+        securityGroupIds:
+        - $brooklyn:component("tf").attributeWhenReady("tf.output.securityGroupId")
+  brooklyn.config:
+    launch.latch: $brooklyn:component("tf").attributeWhenReady("service.isUp")
+    wars.root: http://search.maven.org/remotecontent?filepath=org/apache/brooklyn/example/brooklyn-example-hello-world-webapp/0.9.0/brooklyn-example-hello-world-webapp-0.9.0.war
+
+- type: io.cloudsoft.terraform.TerraformConfiguration
+  id: tf
+  location: localhost
+  brooklyn.config:
+    tf.configuration.contents: |
+        provider "aws" {
+            access_key = "..."
```

Comment:
Is it worth mentioning here and above how to avoid hard-coding credentials?  We could even just link out to https://www.terraform.io/docs/providers/aws/#ec2-role.